docs/index.html

```diff
@@ -6045,58 +6049,61 @@ <h3 class="heading settled" data-level="12.3" id="text-macros"><span class="secn
 The following macros are defined:</p>
    <ul>
     <li data-md>
-     <p><dfn class="dfn-paneled" data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-title"><code>[TITLE]</code></dfn> gives the spec’s full title, as extracted from either the H1 or the spec metadata.</p>
+     <p><dfn data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-bikeshed-documentation"><code>Bikeshed Documentation</code><a class="self-link" href="#macro-bikeshed-documentation"></a></dfn> gives the spec’s full title, as extracted from either the H1 or the spec metadata.</p>
     <li data-md>
-     <p><dfn class="dfn-paneled" data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-h1"><code>[H1]</code></dfn> gives the desired document heading, in case the in-page title is supposed to be different from the <code>&lt;title></code> element value.</p>
+     <p><dfn data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-h1"><code>[H1]</code><a class="self-link" href="#macro-h1"></a></dfn> gives the desired document heading, in case the in-page title is supposed to be different from the <code>&lt;title></code> element value.</p>
     <li data-md>
-     <p><dfn class="dfn-paneled" data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-shortname"><code>[SHORTNAME]</code></dfn> gives the document’s shortname, like "css-cascade".</p>
+     <p><dfn data-dfn-for="macro" data-dfn-type="dfn" data-export id="macro-bikeshed"><code>bikeshed</code><a class="self-link" href="#macro-bikeshed"></a></dfn> gives the document’s shortname, like "css-cascade".</p>
```

Comment:
The HTML-generation process seems to have gone very wrong here — the newly-generated HTML is actually replacing the macros, instead of documenting them. It seems strings surrounded by `[` and `]` are getting replaced even when inside `<code>`, where previously they were left alone.